app/config.js

```diff
@@ -1,5 +1,5 @@
-import convict from 'convict'
-import convictFormatWithValidator from 'convict-format-with-validator'
+const convict = require('convict')
```

Comment:
shall we leave as [ES](https://defra.github.io/software-development-standards/standards/node_standards/#commonjs-vs-es-modules)?

Reply:
seeing as you did all the work in a single commit might be easier to just rebase that out?

Reply:
Agreed, I did think about that but decided against it due to the tests failing to run, jest needs configuring to work with ES modules. I can do that in this PR if you feel strongly about it?